app/code/core/Mage/Sales/Model/Resource/Order/Abstract.php

```diff
@@ -323,7 +323,7 @@ protected function _afterSaveAttribute(Mage_Core_Model_Abstract $object, $attrib
      * Perform actions after object save
      *
      * @param Mage_Core_Model_Abstract $object
-     * @param string $attribute
+     * @param string|string[] $attribute
```

Comment:
We should also add `Mage_Eav_Model_Entity_Attribute_Abstract` since it seems it's an accepted type.